docs/mcr-endpoints-guidance.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Guidance for using the official Microsoft Container Registry (MCR) endpoints
+description: This articles provides information about the official endpoints of Microsoft Container Registry (MCR) and provides guidance how to use the REST and CDN endpoints to pull official Microsoft artifacts around the world.
+ms.topic: guidance
+ms.date: 02/03/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Guidance for using the official Microsoft Container Registry (MCR) endpoints
+
+Microsoft Container Registry (MCR) is an implementation of the [OCI Distribution Specification][oci-spec] which delivers [artifacts][oci-artifacts], such as container images, helm charts and other assets distributed by Microsoft and Azure. The Distribution Spec linked above defines two endpoints:
```

Comment:
Not sure about Microsoft _and_ Azure. Makes it read like they are two separate entities. Maybe Microsoft including Azure?

Reply:
Removed Azure.